mail.md

```diff
@@ -434,7 +438,7 @@ The `withSwiftMessage` method of the `Mailable` base class allows you to registe
     {
         $this->view('emails.orders.shipped');
 
-        $this->withSwiftMessage(function ($message) {
+        $this->withSymfonyMessage(function (Email $message) {
             $message->getHeaders()->addTextHeader(
```

Comment:
Does this syntax still work for adding headers to a Symfony Email message? We will want to update it accordingly if not.

Reply:
addTextHeader still exists in Headers: https://github.com/symfony/mime/blob/dc5de94574db31fbb51d6b7d22e9d34daa2e305d/Header/Headers.php#L119-L125

and Headers is return on getHeaders() in Message (Email is extended from that):
https://github.com/symfony/mime/blob/dc5de94574db31fbb51d6b7d22e9d34daa2e305d/Message.php#L67-L70